tests/test_linux/conftest.py

```diff
@@ -20,6 +16,7 @@
 
 config.nlm_generator = True
 pytest_plugins = 'pytester'
+pytest_plugins = ["pyroute2.fixtures.iproute"]
```

Comment:
pls don't leave `pytester` out of the list

Reply:
oops !